src/phoenix/trace/langchain/instrumentor.py

```diff
@@ -0,0 +1,38 @@
+# Instruments the OpenInferenceTracer for LangChain automatically
+from typing import Any, Optional
+
+from phoenix.trace.instrumentor import Instrumentor
+
+from .tracer import OpenInferenceTracer
+
+MINIMUM_LANGCHAIN_VERSION = "0.13.0"
+
+
+class LangChainInstrumentor(Instrumentor):
+    def __init__(self, tracer: Optional[OpenInferenceTracer] = None) -> None:
+        self._tracer = tracer if tracer is not None else OpenInferenceTracer()
```

Comment:
Probably don't want to lose the `source_init`. Saving it also lets us check whether patching has taken place.
```suggestion
        self._tracer = tracer if tracer is not None else OpenInferenceTracer()
        self._source_init: Any = None
```

Reply:
I added it but not in the constructor since mypi doesn't like the changing of types.